ISISScientificComputing/autoreduce
# Add and integrate Software table

### Summary of work
* Added the Software table to the model
* Added access function for get/create
* Added version find function for mantid
* Propagate version through autoreduction processor to be saved in queue processor

### How to test your work
* Push this code to the dev nodes (will need to perform a database migration) to get the new table
* perform manual submission and ensure that the software records are created and associated with each run

Fixes #528 


**Before merging ensure the release notes have been updated**
